test/auth/auth-manager-test.js

```diff
@@ -0,0 +1,129 @@
+const mockgoose = require('mockgoose');
+const mongoose = require('mongoose');
+const bluebird = require('bluebird');
+const chai = require('chai');
+const chaiMoment = require('chai-moment');
+const jwt = require('jsonwebtoken');
+const authManager = require('../../src/auth/auth-manager');
+const authModels = require('../../src/auth/auth-models');
+const userManager = require('../../src/users/user-manager');
```

Comment:
okay that makes sense. The `user-manager` is the one I was more concerned about.